components/Collections.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button,
+  ButtonGroup,
+  Spinner,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { getServicesList } from 'api'
+import { UserDoc } from 'models/users'
+import React from 'react'
+import { useQuery } from 'react-query'
+import CollectionItem from './CollectionItem'
+
+interface CollectionProps {
+  userData: UserDoc
+}
+
+export const Collections = ({ userData }: CollectionProps): JSX.Element => {
+  const queryFn = async () => {
+    // remove duplicates and create fetch promises
+    const promises = userData.lists.map((list) => getServicesList(list))
+
+    // fetch all lists in parallel and remove ones that errored
+    const result = (
+      await Promise.all<typeof promises>(promises.map((p) => p.catch((e) => e)))
+    ).filter((l) => !(l instanceof Error))
+
+    return result
+  }
+
+  const { data, isLoading } = useQuery({
+    queryKey: ['serviceListsResource', userData.lists],
+    queryFn,
+  })
```

Comment:
I think `useQueries` (like mentioned in another comment) is preferable to wrapping all of the API calls in a single query function.